client/components/tinymce/plugins/media/restrict-size/index.js

```diff
@@ -58,6 +70,15 @@ export default function( editor ) {
 		}
 
 		event.content = setImages( event.content );
+
+		/**
+		 * Also add the raw resized image URL so we can properly preload it in the editor
+		 */
+		const resizedUrl = getResizedImgUrlFromImgString( event.content );
```

Comment:
Do we ever need this to run on `BeforeSetContent`?

Instead of:

```js
editor.on( 'BeforeSetContent BeforeSetWpcomMedia', ( event ) => {
	if ( ! event.content || 'html' === event.mode ) {
		return;
	}

	event.content = setImages( event.content );

	/**
	 * Also add the raw resized image URL so we can properly preload it in the editor
	 */
	const resizedUrl = getResizedImgUrlFromImgString( event.content );

	if ( resizedUrl !== null ) {
		event.resizedImageUrl = resizedUrl.resizedUrl;
	}
} );
```

I'm suggesting:

```js
editor.on( 'BeforeSetContent BeforeSetWpcomMedia', ( event ) => {
	if ( ! event.content || 'html' === event.mode ) {
		return;
	}

	event.content = setImages( event.content );
} );

editor.on( 'BeforeSetWpcomMedia', ( event ) => {
	/**
	 * Also add the raw resized image URL so we can properly preload it in the editor
	 */
	const resizedUrl = getResizedImgUrlFromImgString( event.content );

	if ( resizedUrl !== null ) {
		event.resizedImageUrl = resizedUrl.resizedUrl;
	}
} );
```